src/plugins/Transloadit/index.js

```diff
@@ -65,6 +66,10 @@ module.exports = class Transloadit extends Plugin {
       })
 
       this.core.setState({ files })
+
+      if (this.opts.wait) {
+        return this.beginWaiting()
+      }
     }).catch((err) => {
       this.core.emit('informer', '⚠️ Transloadit: Could not create assembly', 'error', 0)
```

Comment:
Yeah error feedback to the end user needs some work. This happens inside a Promise chain inside the `uppy.upload()` call, I've added an error log to its call site in Dashboard. If developers call `uppy.upload()` manually, they should add their own `.catch` handler.